test/e2e/positive.go

```diff
@@ -50,4 +71,98 @@ var _ = ginkgo.Describe("[PositiveFlow] Deployer validation", func() {
+
+		ginkgo.It("should have API components", func() {
+			mf, err := api.GetManifests()
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+
+			key := getObjKey(mf.Crd.Name, mf.Crd.Namespace)
+			err = hp.WaitForObjectToBeCreated(key, &v1.CustomResourceDefinition{})
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+		})
+
+		ginkgo.It("should have all RTE components", func() {
+			mf, err := rte.GetManifests()
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+			mf = mf.UpdateNamespace()
+
+			for _, obj := range mf.ToObjects() {
+				key := getObjKey(obj.GetName(), obj.GetNamespace())
+				err = hp.WaitForObjectToBeCreated(key, obj)
+				gomega.Expect(err).ToNot(gomega.HaveOccurred(), fmt.Sprintf("object of kind %v was not created", obj.GetObjectKind()))
+			}
+		})
+
+		ginkgo.It("should have SCHED components", func() {
+			mf, err := sched.GetManifests()
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+			mf = mf.UpdateNamespace()
+
+			for _, obj := range mf.ToObjects() {
+				key := getObjKey(obj.GetName(), obj.GetNamespace())
+				err = hp.WaitForObjectToBeCreated(key, obj)
+				gomega.Expect(err).ToNot(gomega.HaveOccurred(), fmt.Sprintf("object of kind %v was not created", obj.GetObjectKind()))
+			}
+
+		})
+
+		ginkgo.It("should have all pods running in all namespaces", func() {
+			ginkgo.By("checking that resource-topology-exporter pod is running")
+			mf, err := rte.GetManifests()
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+			mf = mf.UpdateNamespace()
+
+			found := getPodByRegex(fmt.Sprintf("%s-*", mf.DaemonSet.Name))
+			gomega.Expect(found).To(gomega.BeTrue())
+
+			ginkgo.By("checking that topo-aware-scheduler pod is running")
+			mfs, err := sched.GetManifests()
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+			mfs = mfs.UpdateNamespace()
+
+			found = getPodByRegex(fmt.Sprintf("%s-*", mfs.Deployment.Name))
+			gomega.Expect(found).To(gomega.BeTrue())
+		})
+
+		ginkgo.It("should have some information in noderesourcetopology objects", func() {
+			key := getObjKey(nodeResourceTopology, exportedNs)
+			nrt := &v1alpha1.NodeResourceTopology{}
+			err := hp.WaitForObjectToBeDeleted(key, nrt)
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+
+			// TODO check if specific resources are exist under the noderesourcetopology object
+			})
+	})
 })
+
+func getObjKey(name, ns string) types.NamespacedName {
+	if len(ns) == 0 {
+		return types.NamespacedName{
+			Name: name,
+		}
+	}
+	return types.NamespacedName{
+		Name: name,
+		Namespace: ns,
+	}
+}
+
+func getPodByRegex(reg string) bool {
+	cs, err := clientutil.New()
+	gomega.Expect(err).ToNot(gomega.HaveOccurred())
+
+	podNameRgx, err := regexp.Compile(reg)
+	gomega.Expect(err).ToNot(gomega.HaveOccurred())
+
+	podList := &corev1.PodList{}
+	err = cs.List(context.TODO(), podList)
+	gomega.Expect(err).ToNot(gomega.HaveOccurred())
+	for _, pod := range podList.Items {
+		if match := podNameRgx.FindString(pod.Name); len(match) != 0 {
```

Comment:
but we don't actually check the pods are running (and without errors in their status), do we?

Reply:
You're right noticed that, have it in progress